presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -80,6 +80,9 @@ CHAR
 ----
 
     Fixed length character data. Char type without length specified has default length of 1.
+    Comparison semantics is that of VARCHAR type (leading and trailing spaces are relevant).
```

Comment:
I don't quite understand what this means.  Is this saying that if I have a column with type `CHAR(7)` and I put in the word "dog", that it would be padded with white space up to length 7, so that it would actually be evaluated throughout as "dog    "?  If so talk about the whitespace padding explicitly.  Also, is that just for comparison or for all cases?  If I concatenate "cat" to that CHAR(7) "dog", would that show up as "dogcat" or "dog    cat"?

Here's a rewrite of the bit that you have so far:
Leading and trailing spaces are included in comparisons of CHAR values.  As a result, two character values with different lengths will never be equal.